Kitodo/src/main/java/org/kitodo/workflow/model/beans/KitodoTask.java

```diff
@@ -43,6 +44,7 @@
      *            * determined out of sequence flow
      */
     public KitodoTask(Task task, int ordering) {
+        this.workflowId = task.getId();
```

Comment:
You are sure that this assignment is correct? Should it not be `task.getWorkflowId()`?

Reply:
Yes, it is correct. Task object is a task from BPMN model (not our bean).

Reply:
This is a little bit confusing....

Reply:
I know but I can't change name of the class inside the bpmn model library.